ui/main/src/app/modules/archives/archives.component.html

```diff
@@ -112,18 +112,7 @@
 <div *ngIf="technicalError" style="text-align: center;font-size: 20px;" translate> shared.error.technicalError
 </div>
 
-<div *ngIf="loadingIsTakingMoreThanOneSecond">
-
-  <div class="opfab-card-loading-spinner" id="opfab-loading-spinner">
-  <em  class="fas fa-spinner fa-spin opfab-slow-spinner" ></em>
-  </div>
-  <div style="text-align: center;font-size:16px;font-weight: bold;margin-top:10px" translate>
-      archive.loadingInProgress
-  </div>
-
-  <br/>
-  <br/>
-</div>
+<of-spinner *ngIf="loadingInProgress" loadingText="archive.loadingInProgress"></of-spinner>
```

Comment:
use the generic message here and remove translation key archive.loadingInProgress